.github/workflows/docs.yml

```diff
@@ -20,18 +20,35 @@ jobs:
       - name: Create conda environment
         run: conda create --quiet -c conda-forge --name shpc spython
 
+      - name: Check out main
+        uses: "actions/checkout@v3"
+        with:
+          ref: "main"
+          path: "tmp-registry"
+
       - name: Install shpc
         run: |
           export PATH="/usr/share/miniconda/bin:$PATH"
           source activate shpc
           pip install git+https://github.com/singularityhub/singularity-hpc@main
-          git clone https://github.com/singularityhub/shpc-registry /tmp/registry
+
+      - name: Set Pages base URL and Registry URL
+        run: |
+          URL=$(curl -L \
```

Comment:
I think there is some risk here of running this regularly (and having the API response not return successfully, etc.) I want to suggest we add this instruction to the tutorial, to be done once, and it's OK if it's automated like this, or just a manual instruction.